pkg/cmd/roachtest/cluster_test.go

```diff
@@ -168,6 +168,10 @@ func TestClusterMachineType(t *testing.T) {
 		{"n2-standard-32", 32},
 		{"n2-standard-64", 64},
 		{"n2-standard-96", 96},
+		// GCE machine types
```

Comment:
Stray comment? There is already a comment above for "GCE machine types".

Reply:
Thanks, removing.